presto-product-tests/bin/run_on_docker.sh

```diff
@@ -238,6 +240,9 @@ PRESTO_LOGS_PID=$!
 # wait until presto is started
 retry check_presto
 
+# make sure that hadoop is still alive
+check_hadoop
```

Comment:
without retry this time?

Reply:
yes. We had retry previously, so hadoop should be up and running. No need to wait.